allure-framework-modules/allure-core-module/src/test/java/com/capgemini/ntc/test/core/base/encryption/DataEncryptionTest.java

```diff
@@ -0,0 +1,188 @@
+package com.capgemini.ntc.test.core.base.encryption;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import com.capgemini.ntc.test.core.base.encryption.providers.DataEncryptionService;
+import com.capgemini.ntc.test.core.exceptions.BFSecureModuleException;
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Provides;
+
+public class DataEncryptionTest {
+	
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+		
+	}
+	
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+	
+	private IDataEncryptionService systemUnderTest;
+	
+	@Before
+	public void setUp() throws Exception {
+		
+		systemUnderTest = Guice.createInjector(securedataTestModel())
+		        .getInstance(IDataEncryptionService.class);
+		
+	}
+	
+	@After
+	public void tearDown() throws Exception {
+	}
+	
```

Comment:
@marpuch : Everything looks great. Missing only one positive test: 

`@Test()
     public void dataEncryptions_with_default_secretdata_file() {
IDataEncryptionService systemUnderTest = Guice.createInjector(new DataEncryptionModule())
		        .getInstance(IDataEncryptionService.class);
//Some set of assert/s to validate it works 
}`